packages/chart/src/scss/main.scss

```diff
@@ -14,6 +14,14 @@
 .flex-column-reverse {
   flex-direction: column-reverse;
 }
+.visx-brush {
+  & > rect {
+    stroke-width: 1px !important;
```

Comment:
Do you need to use !important here?
It looks like rx isn't a valid css property, however, removing it breaks the look. Look into workarounds for this. It's also not supported by Firefox.